improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -993,6 +992,8 @@ def _location_and_scale_parameters_to_probabilities(
                 probability_method(threshold), probabilities.shape[1:])
 
         probability_cube = probability_cube_template.copy(data=probabilities)
+        if (np.isnan(probability_cube.data).any()) == True:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comparison to True should be just 'expr'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4922716)